packages/ui/src/molecules/Accordion/AccordionItem.tsx

```diff
@@ -21,17 +21,27 @@ export interface AccordionItemProps extends HTMLAttributes<HTMLDivElement> {
    * Index of the current accordion item within the accordion.
    */
   index?: number
+  /**
+   * ID for the current Accordion item's panel and button
```

Comment:
Maybe my suggestion is too long.

```suggestion
   * Namespace ID prefix for the current Accordion item's panel and button to avoid ID duplication when multiple instances are on the same page.
```